must-gather/gather-kepler-operator-info

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+# copyright 2023.
+#
+# licensed under the apache license, version 2.0 (the "license");
+# you may not use this file except in compliance with the license.
+# you may obtain a copy of the license at
+#
+#     http://www.apache.org/licenses/license-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the license is distributed on an "as is" basis,
+# without warranties or conditions of any kind, either express or implied.
+# see the license for the specific language governing permissions and
+# limitations under the license.
+#
+
+set -eu -o pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source "${SCRIPT_DIR}/common"
+
+BASE_COLLECTION_PATH=$1
+
+KEPLER_OPERATOR_NS="openshift-operators"
+
+get_catalogsource() {
+  log "collecting catalogsource info for kepler-operator"
+  run oc -n "$KEPLER_OPERATOR_NS" get catalogsource kepler-operator-catalog -oyaml "$KEPLER_OPERATOR_INFO_DIR/kepler-operator-catalogsource.yaml"
+}
+
+get_subscription() {
+  log "collecting subscription info for kepler-operator"
+  run oc -n "$KEPLER_OPERATOR_NS" get subscription -l operators.coreos.com/kepler-operator.openshift-operators= -oyaml "$KEPLER_OPERATOR_INFO_DIR/kepler-operator-subscription.yaml"
+}
+
+get_install_plan() {
+  log "collecting installplan info for kepler-operator"
+  run oc -n "$KEPLER_OPERATOR_NS" get installplan -l operators.coreos.com/kepler-operator.openshift-operators= -oyaml "$KEPLER_OPERATOR_INFO_DIR/kepler-operator-installplan.yaml"
+}
+
+get_csv() {
+  log "collecting CSV for kepler-operator"
+  run oc -n "$KEPLER_OPERATOR_NS" get csv -l operators.coreos.com/kepler-operator.openshift-operators= -oyaml "$KEPLER_OPERATOR_INFO_DIR/kepler-operator-csv.yaml"
+}
+
+get_kepler_operator_deployment_info() {
+  KEPLER_OPERATOR_DEPLOY="kepler-operator-controller-manager"
+  log "collecting deployment info for kepler-operator"
+  run oc -n "$KEPLER_OPERATOR_NS" get deployment "$KEPLER_OPERATOR_DEPLOY" -oyaml "$KEPLER_OPERATOR_INFO_DIR/kepler-operator-deployment.yaml"
+}
+
+get_kepler_operator_pod_info() {
+  log "collecting pod info for kepler-operator" 
+  run oc -n "$KEPLER_OPERATOR_NS" get pod -l control-plane=controller-manager -oyaml "$KEPLER_OPERATOR_INFO_DIR/kepler-operator.yaml"
+}
+
+get_summary() {
+  run oc -n "$KEPLER_OPERATOR_NS" get catalogsource kepler-operator-catalog -owide "$KEPLER_OPERATOR_INFO_DIR/summary.txt"
```

Comment:
In the scenario where operator is installed from `community-catalog`, `get_summary` breaks here  and the script terminates execution
```
2023-10-05 01:24:48 oc -n openshift-operators get subscription -l operators.coreos.com/kepler-operator.openshift-operators= -oyaml
2023-10-05 01:24:48 collecting catalogsource info for kepler-operator
2023-10-05 01:24:48 oc -n openshift-operators get catalogsource kepler-operator-catalog -oyaml
Error from server (NotFound): catalogsources.operators.coreos.com "kepler-operator-catalog" not found
~
```